Userland/Libraries/LibSQL/AST/Token.h

```diff
@@ -213,11 +213,13 @@ enum class TokenCategory {
 
 class Token {
 public:
-    Token(TokenType type, StringView value, size_t line_number, size_t line_column)
+    Token(TokenType type, StringView value, size_t line_number, size_t line_column, size_t end_line_number, size_t end_line_column)
```

Comment:
Because we now store start and end line/column, it probably makes sense to also rename `line_number` and `line_column` to `start_line_number` and `start_line_column`. Or, to have some small struct like:
```c++
struct SourcePosition {
    size_t line {0};
    size_t column {0};
};
```

And have the constructor take parameters `SourcePosition start_potion, SourcePosition end_position`.

Reply:
I like the `struct` idea. Will fix.